### DISCUSSION TITLE
compiling error on nrsconfig

### DISCUSSION THREAD

**Discussion by YunchaoYang** [Nek5000/nekRS#434] [Compiling]
Hi,

I am trying to compile nekrs on a supercomputer. I encountered this error on nrsconfig:

CMake Error: Could not open file for write in copy operation /usr/local/nekrs.conf.tmp
CMake Error: : System Error: Permission denied
CMake Error at CMakeLists.txt:330 (configure_file):
  configure_file Problem configuring file

It seems that cmake is trying to access the system folder /user/local, 
but isn't it supposed to use $NEKRS_HOME=$HOME/.local/nekrs ?

Any suggestions on how to resolve this? 

Thanks in advance
.

**Comment by stgeke** [comment]
I cannot reproduce it on my end. Can you please outline (step-by-step) what you did.

**Reply by YunchaoYang** [reply]
Thanks for the response. All I did just is "git clone" and run ./nrsconfig directly. 

Actually this problem is solved. I was trying to traceback the error message from CMakeLists.txt line 330. It turns out that the variable "CMAKE_INSTALL_PREFIX" was set to the default of /usr/local. I added this line `message(STATUS "CMAKE_INSTALL_PREFIX: ${CMAKE_INSTALL_PREFIX}")` in CMakeLists.txt to print out the variable "CMAKE_INSTALL_PREFIX" and rerun ./nrsconfig. The error message is just gone and "CMAKE_INSTALL_PREFIX" is set correctly.

**Comment by stgeke** [comment]
That's odd because we set `CMAKE_INSTALL_PREFIX` in `nrsconfig`.
